### DISCUSSION TITLE
No invoke of callback function while other dialog moves

### DISCUSSION THREAD

**Discussion by skubidu1324** [MicrosoftEdge/WebView2Feedback#4030] [Q&A]
We have an application in which we can open a dialog containing a WebView2. Now we have the strange behavior that if you move another dialog (with or without WebView2) while the Webview2 initializes the controller or starts a URL via `NavigateStarting` the Webview2 no longer invokes its callback function (`CoreWebView2InitializationCompleted` or `NavigationCompleted`).

We have already tried to turn on logging with
`--enable-logging --v=1 --log-file=c:\temp\webview.log`

and compare the logging with and without moving the other dialog but without any success.

Pausing and looking at the call stack didn't bring any results either because there was no differences. How can we debug the WebView2 to find out what exactly is causing the error that the WebView2 not invokes the callback? Each additional WebView2 can't be initialized, so that only a restart of the entire application helps.

Edit:
• Framework: MFC application which calls a WinForms dialog
• SDK and Runtime version: FixedVersionRuntime.108.0.1462.76.x86 and FixedVersionRuntime.110.0.1587.56.x86
• Operating System: Windows (7,10,11)

**Comment by novac42** [comment]
Hi could you please provide some more information about this?

	• Framework (WinUI, Win32, WinForms, UWP, WPF, etc)
	• SDK and Runtime version.
	• Operating System

**Comment by skubidu1324** [comment]
1. Start first dialog. Logging looks good.
![grafik](https://user-images.githubusercontent.com/68343270/224911633-55105f05-cfe9-4782-8add-46db70834dc0.png)
2. Start second dialog and moving first dialog while navigation. No callback `NavigationCompleted`
![grafik](https://user-images.githubusercontent.com/68343270/224911809-d7e2e9e9-2aeb-4822-b3d3-4d66dfce51a9.png)
3. Start third dialog. Nothing works, not even the initialization of the controller 
![grafik](https://user-images.githubusercontent.com/68343270/224912232-e6a3f81b-9b56-49ec-a340-9484872bf690.png)

If I start the three dialogs without moving one everthing works fine.
![grafik](https://user-images.githubusercontent.com/68343270/224912781-8298d3af-4d85-4bc5-9f56-3d31a90de212.png)

**Comment by ghost** [comment]
Hi @skubidu1324 , apologies for the delay in responding to this. Could you provide us with either a repro app or a dump of the app and browser process when you are experiencing this so we can take a look?

**Comment by skubidu1324** [comment]
Thank you for your reply. Do you have an email address that I can send the dumps to?

**Comment by skubidu1324** [comment]
> Hi @skubidu1324 , apologies for the delay in responding to this. Could you provide us with either a repro app or a dump of the app and browser process when you are experiencing this so we can take a look?

@yildirimcagri : Something I can do? Where can I send the dumps to?
